plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -399,6 +425,7 @@ def main():
         ],
         required_together=[
             ['api_username', 'api_password'],
+            ['default_branch', 'initialize_with_readme']
```

Comment:
```suggestion
```
This forces users to always specify `default_branch` when they specify `initialize_with_readme`. This would make this PR a breaking change.

I don't think this condition can be checked with the existing parameters for `AnsibleModule`, so you either have to do it manually, or not do it at all.

Reply:
Yes you're right, i add this instead:
if default_branch and not initialize_with_readme:
        module.fail_json(msg="Param default_branch need param initialize_with_readme set to true")